src/Rector/Contrib/PHPUnit/SpecificMethod/AssertSameBoolNullToSpecificMethodRector.php

```diff
@@ -11,14 +11,10 @@
 
 /**
  * Before:
- * - $this->assertSame(null, $anything);
- * - $this->assertSame(true, $anything);
- * - $this->assertSame(false, $anything);
+ * - $this->assertSame({keyword}}, $anything);
  *
  * After:
- * - $this->assertNull($anything);
- * - $this->assertTrue($anything);
- * - $this->assertFalse($anything);
+ * - $this->assert{keyword}($anything);
```

Comment:
This could mean that any `assert*` function is changed, which is not true.
Only these 3 method are touched and it's even in class name

Reply:
Gonna fix them in #214!